### PR TITLE
get libsodium from package root

### PR DESCRIPTION
**Pull request by maasteh**

### CODE REVIEW

File: libnacl/__init__.py [hunk 1]
```diff
@@ -12,21 +12,24 @@
...
+    # attempt to import local dll
+    if not sys.platform.startswith("linux"):
+        name = 'libsodium' + lib_ext()
         try:
-            return ctypes.cdll.LoadLibrary(env_libsodium_path)
-        except OSError:
-            raise OSError("Could not find dynamic lib at LIBSODIUM_PATH=%s" % env_libsodium_path)
+            return ctypes.cdll.LoadLibrary(os.path.join(os.path.dirname(os.path.abspath(__file__)), name))
+        except OSError as e: # fall back to default
+            print e
```
Comment (ianyh) [inline libnacl/__init__.py:32]: Is printing here actually meaningful?

**Review by ianyh — COMMENTED** [review]
Why the change?

**Comment by maasteh** [comment]
> Why the change?

client depends on this dll to be installed. It's more reasonable to internalize it as a module rather than having it as a step in environment initialization.

**Comment by samantehrani** [comment]
yes, it would be helpful to see what is the issue, if any, in stdout

File: libnacl/__init__.py (hunk 1 above)
Comment (maasteh) [reply libnacl/__init__.py:32]: yes, it would be helpful to see what is the issue, if any, in stdout

**Review by ianyh — APPROVED** [review]

**Pull request merged**